code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -98,6 +98,12 @@
 /turf/open/proc/update_visuals()
 	var/list/new_overlay_types = tile_graphic()
```

Comment:
Inline this proc please

Reply:
the point is to avoid changes until we're actualy using 512 full time

Reply:
k